### PR TITLE
feat: create login and registration screens

### PR DESCRIPTION
**Pull request by Darshan6069**
create loginscreen : https://www.figma.com/design/PJGuzcXrBGIzhFvXqdmYWl/Rnw-Food-Delivery?node-id=635-11109&t=Ks8hd2UkhfDFdici-4

registration Screen: https://www.figma.com/design/PJGuzcXrBGIzhFvXqdmYWl/Rnw-Food-Delivery?node-id=635-11183&t=Ks8hd2UkhfDFdici-4

### CODE REVIEW

File: assets/AuthImages/icons8-apple-50.png
Comment (nikkielevancex) [inline assets/AuthImages/icons8-apple-50.png:1]: **suggestion**: replace all pngs with vector svgs

File: lib/core/constant/strings.dart [hunk 1]
```diff
@@ -1,27 +1,39 @@
-class IntroScreen{
-  static const title = 'We serve incomparable delicacies';
+ class IntroScreenString{
+  static const title1 = '   We server \nincomparable \n   delicacies';
+  static const title2 = '   Find the \n     Best \nRestaurants';
+  static const title3 = ' They can\'t \n   wait for \nyour order!!';
   static const subtitle = 'All the best restaurants with their top menu waiting for you, they cant’t wait for your order!!';
 }
 
-class RegistrationScreen{
```
Comment (nikkielevancex) [inline lib/core/constant/strings.dart:6]: **suggestion**: take all strings under one class called AppStrings

File: lib/Domain/OnboardingModel/onboarding_model.dart [hunk 2]
```diff
@@ -10,21 +12,21 @@ class OnboardingModel {
 List<OnboardingModel> onboardingDetailsList=[
   OnboardingModel(
     image: 'assets/OnboardingImage/onboardingImage1.png',
-    title: '   We server \nincomparable \n   delicacies',
+    title: IntroScreenString.title1,
     description:
     'All the best restaurants with their top menu waiting for you, they cant’t wait for your order!!',
```
Comment (nikkielevancex) [inline lib/Domain/OnboardingModel/onboarding_model.dart:17]: **suggestion**: create a const variable and reuse it in call models

File: lib/Presentation/Authentication/login_screen.dart [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+              padding: EdgeInsets.all(context.height(context) * 0.030),
+              child: const Row(
+                 mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  CommonCircleButton(
+                      image: ImagePath.googleImage),
+                  CommonCircleButton(
+                      image: ImagePath.facebookImage),
+                  CommonCircleButton(
+                      image: ImagePath.appleImage),
```
Comment (nikkielevancex) [inline lib/Presentation/Authentication/login_screen.dart]: **suggestion**: use list.map

File: lib/Presentation/Authentication/login_screen.dart [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            SizedBox(height:context.height(context) * 0.030),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                const Text(LoginScreenString.notHaveAccount,
+                    style: TextStyle(fontSize: 16)),
+                GestureDetector(
+                  onTap: () {
+                    context.pushAndRemoveUntil(context, target: const RegistrationScreen());
+                  },
```
Comment (nikkielevancex) [inline lib/Presentation/Authentication/login_screen.dart]: **suggestion**: use tearoff

File: lib/Presentation/Authentication/registration_screen.dart [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+              padding: EdgeInsets.all(context.height(context) * 0.008),
+              child: Row(
+                children: [
+                  Checkbox(
+                    value: true,
+                    onChanged: (value) {},
+                  ),
+                  RichText(
+                    text: const TextSpan(
+                      text: 'I Agree with ',
```
Comment (nikkielevancex) [inline lib/Presentation/Authentication/registration_screen.dart:75]: **suggestion**: use from AppStrings

File: assets/AuthImages/icons8-apple-50.png
Comment (Darshan6069) [reply assets/AuthImages/icons8-apple-50.png:1]: change png image to svg

**Review by nikkielevancex — APPROVED** [review]

**Pull request merged**